blog/2024-07-09-fan-fanentityfeatures-turn-on_off.md

```diff
@@ -0,0 +1,17 @@
+---
+author: G Johansson
+authorURL: https://github.com/gjohansson-ST
+title: "New entity feature flags in FanEntity"
+---
+
+As of Home Assistant Core 2024.8 we have added two new flags into `FanEntityFeature`: `TURN_ON`, `TURN_OFF`.
+
+Integrations implementing `turn_on` service call needs to set the `TURN_ON` feature flag.
+Integrations implementing `turn_off` service call needs to set the `TURN_OFF` feature flag.
+
+There will be a 6 month deprecation period (2025.2) where `FanEntity` will set these on behalf of the integrations implementing the respective methods and from 2025.2 it will make integrations unable to use the respective methods if entity features has not been set accordingly.
```

Comment:
```suggestion
There will be a 6-month deprecation period (2025.2) where `FanEntity` will set these on behalf of the integrations implementing the respective methods and from 2025.2 it will make integrations unable to use the respective methods if entity features have not been set accordingly.
```